gallery/src/styles/NxFormLayout/NxFormLayoutPage.tsx

```diff
@@ -21,128 +21,92 @@ const NxFormLayoutPage = () =>
+              </NxTable.Cell>
+              <NxTable.Cell>Top-Level alternative to <NxCode>NxForm</NxCode></NxTable.Cell>
+              <NxTable.Cell>
+                An alternative to <NxCode>NxForm</NxCode> that internalizes the logic around when to show form-wide
+                validation errors
+              </NxTable.Cell>
+            </NxTable.Row>
+            <NxTable.Row>
+              <NxTable.Cell>
+                <NxCode><NxTextLink href="#/pages/Form Group">NxFormGroup</NxTextLink></NxCode>
+              </NxTable.Cell>
+              <NxTable.Cell>
+                Descendant of <NxCode>NxForm</NxCode>, potentially wrapped in <NxCode>NxFormRow</NxCode>
+              </NxTable.Cell>
+              <NxTable.Cell>
+                Wrapper for an individual form control which gives it a label and optional sublabel.
+              </NxTable.Cell>
+            </NxTable.Row>
+            <NxTable.Row>
+              <NxTable.Cell><NxCode>NxFormRow</NxCode></NxTable.Cell>
+              <NxTable.Cell>Descendant of <NxCode>NxForm</NxCode></NxTable.Cell>
+              <NxTable.Cell>
+                Container for multiple <NxCode>NxFormGroup</NxCode> or <NxCode>NxButtonBar</NxCode>s which causes them
+                to lay out horizontally rather than stack vertically.
+              </NxTable.Cell>
+            </NxTable.Row>
+            <NxTable.Row>
+              <NxTable.Cell><NxCode><NxTextLink href="#/pages/Fieldset">NxFieldset</NxTextLink></NxCode></NxTable.Cell>
+              <NxTable.Cell>Descendant of <NxCode>NxForm</NxCode></NxTable.Cell>
+              <NxTable.Cell>
+                A fieldset styled similarly to <NxCode>NxFormGroup</NxCode>, intended as a wrapper for groups of
+                <NxCode>NxCheckbox</NxCode>es, <NxCode>NxRadio</NxCode>s, or <NxCode>NxToggle</NxCode>s.
+              </NxTable.Cell>
+            </NxTable.Row>
+            <NxTable.Row>
+              <NxTable.Cell><NxCode><NxTextLink href="#/pages/Read Only">NxReadOnly</NxTextLink></NxCode></NxTable.Cell>
+              <NxTable.Cell>Descendant of <NxCode>NxForm</NxCode></NxTable.Cell>
+              <NxTable.Cell>
+                Static/read-only data presented in a format that matches the interactive parts of the form: the data
+                labels are styled equivalently to <NxCode>NxFormGroup</NxCode>'s labels, and the data itself takes
+                the place of a form control such as <NxCode>NxTextInput</NxCode>.
+              </NxTable.Cell>
+            </NxTable.Row>
+            <NxTable.Row>
+              <NxTable.Cell><NxCode>NxFooter</NxCode></NxTable.Cell>
+              <NxTable.Cell>Part of <NxCode>NxForm</NxCode>, not used directly by calling code</NxTable.Cell>
+              <NxTable.Cell>
+                <NxCode>NxForm</NxCode> renders a footer element which contains submission and form-wide validation
```

Comment:
```suggestion
                <NxCode>NxForm</NxCode> renders a footer element which contains form-wide validation
```

Reply:
No there's still submission errors as well, separately from everything going on in this feature branch